src/Compilers/Core/Portable/Emit/EditAndContinue/ILambdaHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Emit
+{
+    internal interface ILambdaSyntaxHelper
```

Comment:
I've done the rename.  I can't change the existing SyntaxFacts as it's a public API already.